src/main/java/com/example/spring/models/Book.java

```diff
@@ -0,0 +1,25 @@
+package com.example.spring.models;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Table(name = "books")
+@Data
```

Comment:
```suggestion
@Getter
@Setter
```

don't use `@Data` for entities